client/src/components/application/NonlinearStepper/GeneralInfo.js

```diff
@@ -0,0 +1,106 @@
+import React from 'react';
+import { Radio, FormControlLabel, RadioGroup } from '@material-ui/core';
+import PropTypes from 'prop-types';
+import InputText from '../../common/TextField';
+import SelectBox from '../SelectBox';
+import Typography from '../../common/Typography';
+import { GeneralInfoStyles } from './style';
+
+const GereralInfos = ({ handleFormInput, formValues, errorMsg }) => {
+  const classes = GeneralInfoStyles();
+
+  return (
+    <>
+      <div className={classes.gender}>
+        <Typography variant="h5" color="primary">
+          Gender
+        </Typography>
+        <RadioGroup
+          row
+          aria-label="gender"
+          name="gender"
+          onChange={handleFormInput}
+          value={formValues.gender}
+          defaultValue="Female"
```

Comment:
default value didn't set , its empty and there is no default value, please check it

Reply:
I will delete it, I think we don't need a default value